plutus-benchmark/script-contexts/test/Data/9.6/checkScriptContext1-20.eval.golden

```diff
@@ -0,0 +1 @@
+(constr 0)
```

Comment:
This is correct.

TODO: write tests which check that the SOP and the Data versions eval to the same result